From f2cfa9b2d9f97b4322bf6b4a38c7bd59e6a446c9 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Sun, 21 Jan 2024 13:12:27 +0100
Subject: [PATCH] Match type of action_toggle_search_cb to its use

This avoids an incompatible-pointer-types error and a build failure
with GCC 14.
---
 src/font-view-window.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/font-view-window.c b/src/font-view-window.c
index 639ac33..24461f7 100644
--- a/src/font-view-window.c
+++ b/src/font-view-window.c
@@ -758,8 +758,11 @@ font_view_window_show_overview (FontViewWindow *self)
 }
 
 static void
-action_toggle_search_cb (FontViewWindow *self)
+action_toggle_search_cb (GtkWidget  *widget,
+                         const char *action_name,
+                         GVariant   *parameter)
 {
+  FontViewWindow *self = FONT_VIEW_WINDOW (widget);
 
  gtk_toggle_button_set_active (self->search_button,
                                !gtk_toggle_button_get_active (self->search_button));
-- 
GitLab

